src/languages/lsl.js

```diff
@@ -0,0 +1,90 @@
+/*
+Language: Linden Scripting Language
+Description: The Linden Scripting Language is used in Second Life by Linden Labs.
+Author: Builder's Brewery <buildersbrewery@gmail.com>
+Category: scripting
+*/
+
+function(hljs) {
+
+    var LSL_STRING_ESCAPE_CHARS = {
+        className: 'subst',
+        begin: '\\[\\\\nt"\\]'
```

Comment:
Actually, it still makes sense to use the universal `hljs.BACKSLASH_ESCAPE` because it will properly highlight those four escapes which can legally be in an LSL string, while others simply won't be found in LSL code. It's a dirty secret of syntax highlighters: they don't need to actually check syntax, they can assume it's correct :-)

You regex is fine as well except that if you use a string instead of a literal regex syntax, you should double all the back-slashes, so either `/\\[tn"\\]/` or `'\\\\[tn"\\\\]'`.